### ISSUE TITLE
Question fashion miniest 도와주세요!!

### ISSUE THREAD

**Issue by wjdgkals23** [hunkim/DeepLearningZeroToAll#198]
import tensorflow as tf
import random
import matplotlib.pyplot as plt
import numpy as np

tf.set_random_seed(777)

def MinMaxScaler(data):
    numerator = data - np.min(data, 0)
    denominator = np.max(data, 0) - np.min(data, 0)
    # noise term prevents the zero division
    return numerator / (denominator + 1e-7)

training_epochs = 19
batch_size = 50

xy = np.loadtxt('train.csv', delimiter=',', dtype=np.float32) #read training data set
xy2 = np.loadtxt('test.csv', delimiter=',', dtype=np.float32) #read test data set

train_y_batch, train_x_batch = \
    tf.train.batch([xy[0], xy[1:]], batch_size=50)

train_x_batch2, train_y_batch2 = \
    tf.train.batch([xy2[1:], xy2[0]], batch_size=50)

#print(x_data.shape, y_data.shape) check data shape

nb_classes= 10 #0-9 labels

X = tf.placeholder(tf.float32, [None, 784])
Y = tf.placeholder(tf.float32, [None, nb_classes])

W = tf.Variable(tf.random_normal([784, nb_classes]), name='weight')
b = tf.Variable(tf.random_normal([nb_classes]), name='bias')

hypothesis = tf.nn.softmax(tf.matmul(X, W) + b)# made hypothesis using softmax

cost = tf.reduce_mean(-tf.reduce_sum(Y * tf.log(hypothesis), axis=1))
optimizer = tf.train.GradientDescentOptimizer(learning_rate=0.1).minimize(cost)

# Test model
is_correct = tf.equal(tf.arg_max(hypothesis, 1), tf.arg_max(Y, 1))
# Calculate accuracy
accuracy = tf.reduce_mean(tf.cast(is_correct, tf.float32))

with tf.Session() as sess:
    #initialize tensorflow variables
    sess.run(tf.global_variables_initializer())
    #Trianing
    for epoch in range(training_epochs):
        avg_cost = 0
        total_batch = int(950 / 50)
        
        for i in range(total_batch):
            batch_xs, batch_ys = sess.run([train_x_batch, train_y_batch])
            c, _ = sess.run([cost, optimizer], feed_dict={
                            X: batch_xs, Y: batch_ys})
            avg_cost += c / total_batch
            
        print('Epoch:', '%04d' % (epoch + 1),
              'cost =', '{:.9f}'.format(avg_cost))

    print("Learning finished")
    
    batch_xs2, batch_ys2 = sess.run([train_x_batch2, train_y_batch2])
    acc = accuracy.eval(session=sess, feed_dict={X:batch_xs2 , Y:batch_ys2})
    print("%f", acc)
    

이 코드 디버그 돌려보았는데 오류는 뜨지 않는데 
    batch_xs, batch_ys = sess.run([train_x_batch, train_y_batch])
            c, _ = sess.run([cost, optimizer], feed_dict={
                            X: batch_xs, Y: batch_ys})
이 부분에서 sess.run하는 줄에서 죽어버립니다. 무슨이유 인가요???
